bin/proving-service/README.md

```diff
@@ -193,34 +193,50 @@ The status check returns:
 
 ### Proxy Status Endpoint
 
-The proxy service exposes a status endpoint that provides information about the current state of the proxy and its workers. This endpoint can be accessed at `http://<proxy_host>:<status_port>/status`.
+The proxy service exposes a gRPC status endpoint that provides information about the current state of the proxy and its workers. This endpoint implements the `ProxyStatusApi` service defined in `proxy_status.proto`.
 
-The status endpoint returns a JSON response with the following information:
+#### gRPC Service Definition
+
+The status service provides the following method:
+- `Status(ProxyStatusRequest) -> ProxyStatusResponse`: Returns the current status of the proxy and all its workers
+
+#### Response Format
+
+The gRPC response includes the following information:
 - `version`: The version of the proxy
-- `supported_proof_type`: The types of proof that the proxy supports
-- `workers`: A list of workers with their status
+- `supported_proof_type`: The type of proof that the proxy supports (`TRANSACTION`, `BATCH`, or `BLOCK`)
+- `workers`: A list of workers with their status information
+
+Each worker status includes:
+- `address`: The worker's network address
+- `version`: The worker's version
+- `status`: The worker's health status (`UNKNOWN`, `HEALTHY`, or `UNHEALTHY`)
+
+#### Example Usage
+
+You can query the status endpoint using a gRPC client. For example, using `grpcurl`:
+
+```bash
+grpcurl -plaintext -import-path ./proto -proto proxy_status.proto \
+  -d '{}' localhost:8084 proxy_status.ProxyStatusApi.Status
+```
 
 Example response:
```

Comment:
When I run the example I don't see an empty `"workers"` or `"supported_proof_type...` which I would expect:
```
➜ grpcurl -plaintext -import-path ./proto -proto proxy_status.proto \
  -d '{}' localhost:8084 proxy_status.ProxyStatusApi.Status
{
  "version": "0.10.0"
}
```

Reply:
Even if adding a new worker to the proxy, the grpcurl command doesn't show the `"supported_proof_type"` field

Reply:
TIL: `grpcurl` doesn't show default values (empty string, enums set to the first value, etc). Adding `-emit-defaults` prints all values:

```sh
$ grpcurl -plaintext -emit-defaults \
  -import-path ./proto -proto proxy_status.proto \
  -d '{}' localhost:8084 proxy_status.ProxyStatusApi.Status

{
  "version": "0.10.0",
  "supportedProofType": "TRANSACTION",
  "workers": []
}
```